### DISCUSSION TITLE
新增doris的FE节点时日志出现：failed to get fe node type from helper node: HostInfo

### DISCUSSION THREAD

**Discussion by dzqjk** [apache/doris#27419] [A - General / Q&A]
使用doris-2.0.2-rc05分支的编译的二进制包，安装之后进行FE的扩容，新的FE节点使用命令 start_fe.sh --helper host:port --daemon  启动时会出现无法访问辅助节点的错误日志
failed to get fe node type from helper node: HostInfo，我使用了telnet排除了端口是否可以到达的问题，两个节点也是可以正常远程访问的

**Comment by dzqjk** [comment]
希望能够得到解决

**Reply by LemonLiTree** [reply]
在master执行 ALTER SYSTEM ADD FOLLOWER "follower_host:edit_log_port"了吗

**Reply by dzqjk** [reply]
是的，执行了

**Reply by dzqjk** [reply]
执行之后在flow节点使用start_fe.sh --helper master:port --daemon会发现日志中出现无法向master节点请求数据，在mysql client中执行show fronteds\G会发现flow节点的errmsg显示拒绝连接的信息

**Reply by LemonLiTree** [reply]
先start_fe.sh 再ALTER SYSTEM ADD FOLLOWER

**Reply by dzqjk** [reply]
已解决

**Comment by dzqjk** [comment]
解决方法如下述步骤：添加节点是先在master的mysql client中执行alter system add follower "follower_host:edit_log_port",一定得是IP，原来使用的主机名称不能解析为ip再连接，需要指定ip，然后在follower节点执行start_fe.sh --helper master_ip:edit_log_port --daemon,这里也必须是IP，否则无法正常向主节点获取元数据
